Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -219,6 +296,79 @@ private JiraRequestContext GetJiraRequestContext(long instanceId)
             }
         }
 
+        private Dictionary<long, List<DashboardWorklog>> GetLastWorklogsByUsers(List<long> ids, long instanceId)
```

Comment:
What does last worklogs mean?

Reply:
The worklogs from the last 7 days, or whatever the number of days we choose. for now, it's the last 7 days.